docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,256 @@ sidebar_position: 2
+- #### `capabilities` <- Needs a fact check
+   An array of one or more `AuthSig`. The `capabilities` authorize the `AuthSig` address(es) to utilize the resources specified. in the `resourceAbilityRequests`
+- #### `issuedAt`
+   The time the session signature was issued.
+- #### `expiration`
+   The time the session signature becomes invalid.
+- #### `nodeAddress`
+   The specific URL the session signature is meant for.
+
+### `address`
+The session key pair public key.
+
+### `algo`
+The signing algorithm used to generate the session signature.
+
+## Example Implementation
+
+Now that we know what the `getSessionSigs` function does, it's parameters, and it's return values, let's dig into a complete implementation.
+
+The full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/tree/master/session-signatures/getSessionSigs). 
+
+### Installing the Required Dependencies
+<Tabs
+defaultValue="npm"
+values={[
+{label: 'npm', value: 'npm'},
+{label: 'yarn', value: 'yarn'},
+]}>
+<TabItem value="npm">
+
+```bash
+npm install \
+@lit-protocol/auth-helpers \
+@lit-protocol/constants \
+@lit-protocol/lit-node-client \
+node-localstorage \
+ethers@v5
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+yarn add \
+@lit-protocol/auth-helpers \
+@lit-protocol/constants \
+@lit-protocol/lit-node-client \
+node-localstorage \
+ethers@v5
+```
+
+</TabItem>
+</Tabs>
+
+
+
+### Instantiating an Ethers Signer
+The `ETHEREUM_PRIVATE_KEY` environment variable is required.
+```ts
+import { LIT_RPC } from "@lit-protocol/constants";
+import * as ethers from "ethers";
+
+const ethersSigner = new ethers.Wallet(
+  ETHEREUM_PRIVATE_KEY,
+  new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+);
+```
+
+### Instantiating a `LitNodeClient`
+Here we are instantiating an instance of `LitNodeClient` and connecting it to the `datil-dev` Lit network. To simulate a Node.js environment, we use `LocalStorage` from the `node-localstorage` module to store our session keys and metadata.
+
+```ts
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LitNetwork } from "@lit-protocol/constants";
+import { LocalStorage } from "node-localstorage";
+
+let litNodeClient: LitNodeClient;
+
+litNodeClient = new LitNodeClient({
+      litNetwork: LitNetwork.DatilDev,
+      debug: false,
+      storageProvider: {
+        provider: new LocalStorage("./lit_storage.db"),
+      },
+    });
+await litNodeClient.connect();
+```
+
+### Generating Session Signatures
+The session signatures generated in this example provide unrestricted access to all resources. However, in real-world applications, it's more common and secure to limit access. Typically, you would use the specific hash of the encrypted access control condition resource key, granting access only to particular resources rather than everything. To this resource, we then give it the ability to decrypt an encryption access control condition. 
```

Comment:
```suggestion
In this example, we're granting the capability to request to decrypt any data that we may be authorized to decrypt (i.e. we satisfy the Access Control Conditions the data was encrypted with). We could, however, specify the [LitAccessControlConditionResource](https://v6-api-doc-lit-js-sdk.vercel.app/classes/auth_helpers_src.LitAccessControlConditionResource.html) for specific encrypted data we're permitting the decryption capability for. In real-world applications, it's more common and secure to limit access to specific Lit resources instead of specifying the wildcard (`"*"`) identifier.

To get the Lit resource identifier for other resources, you can use the other methods included in [@lit-protocol/auth-helpers](https://v6-api-doc-lit-js-sdk.vercel.app/modules/auth_helpers_src.html) package.
```